keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -513,6 +513,17 @@ kpxc.prepareCredentials = async function() {
 
     kpxc.initLoginPopup();
     kpxc.initAutocomplete();
+
+    if (kpxc.settings.autoFillRelevantCredential) {
+        const pageUuid = await sendMessage('page_get_login_id');
+        if (pageUuid) {
+            const relevantCredential = kpxc.credentials.find(c => c.uuid === pageUuid);
+            if (relevantCredential) {
+                const index = kpxc.combinations.length - 1;
```

Comment:
I'd probably change this to something like:
```
const relevantCredential = kpxc.credentials.find(c => c.uuid === pageUuid);
const combination = kpxc.combinations?.at(-1);
if (relevantCredential && combination) ...
```

Reply:
fixed in [49c25f7](https://github.com/keepassxreboot/keepassxc-browser/pull/2409/commits/49c25f7711e15cca63a4ced3a8127ff80fae4875)